### ISSUE TITLE
Implement backpropagation 

### ISSUE THREAD

**Issue by jacanchaplais** [jacanchaplais/cluster_gnn#3]
Write a script to train the network.

- [ ] use binary cross entropy loss
- [ ] use Adam as optimiser
- [ ] set ROC threshold to 0.5
